src/main/java/org/nuxeo/segment/io/SegmentIOIntegrations.java

```diff
@@ -0,0 +1,20 @@
+package org.nuxeo.segment.io;
```

Comment:
Please add the missing license, and move the author info from Javadoc to license.